README.rst

```diff
@@ -33,7 +33,7 @@ Requirements
 ------------
 
 * Redis.
-* Python 2.6, 2.7 or 3.3.
+* Python 2.7 or 3.3+.
```

Comment:
I wonder if we want to drop this. Maybe we can add it to the tox python builds. Did you check to see if python 2.6 works out of the box?